README.md

```diff
@@ -573,6 +571,11 @@ OPTIONS
       the latest known version.
                            This parameter is used only when the installer is 'olm'.
 
+  --[no-]suggested-namespace
+      Indicate to deploy Eclipse Che in OLM suggested namespace: 'eclipse-che'.
+                           Flag 'chenamespace' is ignored in this case
+                           This parameter is used only when the installer is 'olm'.
+
```

Comment:
@flacatus can we add the docs for the  `--cluster-monitoring` flag?

Reply:
or it is not supposed to be documented due to the fact that it is crwctl feature?

Reply:
@ibuziuk will be a crwctl feature. Will be appear in crwctl README

Reply:
shouldn't it be `olm-suggested-namespace`? Otherwise it may be interpreted as `(chectl) suggested namespace` (i.e. the default namespace) and that's the opposite this flags does (override default namespace with olm suggested one).

Reply:
@flacatus Please, update the flag to `olm-suggested-namespace`